lms/templates/fields/field_readonly.underscore

```diff
@@ -1,10 +1,8 @@
 <label class="u-field-title" aria-hidden="true">
     <%- title %>
 </label>
-<span class="sr" for="u-field-input-<%- id %>"><%- screenReaderTitle %></span>
-<span class="u-field-value">
-    <input id="u-field-input-<%- id %>" aria-describedby="u-field-message-<%- id %>" type="text" name="input" readonly=true value="<%- value %>">
-</span>
+<span class="sr" for="u-field-value-<%- id %>"><%- screenReaderTitle %></span>
+<span class="u-field-value" id="u-field-value-<%- id %>" aria-describedby="u-field-message-<%- id %>"><%- value %></span>
```

Comment:
This whole field is read-only, but it can still have a message (which is the case on the account settings page, where  description of the username appears to the right). And if a notification is shown (all through JS), then instead of a message the u-field-message-notification span will have text. So it is difficult to know when rendering the HTML whether or not to add aria-described-by. How terrible is it if it points to a span that happens to have no text?